app/src/main/java/de/danoeh/antennapod/ui/screen/playback/audio/AudioPlayerFragment.java

```diff
@@ -308,6 +308,9 @@ private void updateUi(Playable media) {
     public void sleepTimerUpdate(SleepTimerUpdatedEvent event) {
         if (event.isCancelled() || event.wasJustEnabled()) {
             AudioPlayerFragment.this.loadMediaInfo(false);
+        } else if (event.isOver()) {
+            toolbar.getMenu().findItem(R.id.set_sleeptimer_item).setVisible(true);
+            toolbar.getMenu().findItem(R.id.disable_sleeptimer_item).setVisible(false);
```

Comment:
Would calling loadMediaInfo like in the cases above work as well? Then there is only one place that changes the toolbar.

Reply:
Will look into it.
This is an old MR that I've split, I think when i created the original code it was done similarly in the above case too.